tests/Unit/Domain.Linnapps.Tests/SernosNoteSpecs/WhenUpdatingSernosNoteWithInvalidNote.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Linn.Products.Domain.Linnapps.Tests.SernosNoteSpecs
+{
+    using System;
+
+    using FluentAssertions;
+
+    using Linn.Common.Domain.Exceptions;
+
+    using NUnit.Framework;
+
+    public class WhenUpdatingSernosNoteWithInvalidNote : ContextBase
+    {
+        private Action action;
+
+        [SetUp]
+        public void SetUp()
+        {
+            this.action = () => this.Sut.Update(string.Empty,"new sernos group", 2, 2, "new trans code");
```

Comment:
what happens if the serial number is empty here? would be good to test if you are allowing for it